cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/log/LogHandlerTestRun.java

```diff
@@ -80,8 +86,10 @@ public void testServiceNext() throws Exception {
     testNextFilter("testApp4", "services", "testService1", MockLogReader.TEST_NAMESPACE);
     testNextNoFrom("testApp4", "services", "testService1", MockLogReader.TEST_NAMESPACE);
     testNext("testApp4", "services", "testService1", false, MockLogReader.TEST_NAMESPACE);
-    testNextRunId("testApp4", "services", "testService1", MockLogReader.TEST_NAMESPACE, "text");
-    testNextRunId("testApp4", "services", "testService1", MockLogReader.TEST_NAMESPACE, "json");
+    testNextRunId("testApp4", "services", "testService1", MockLogReader.TEST_NAMESPACE, "text",
+                  ImmutableList.<String>of());
+    testNextRunId("testApp4", "services", "testService1", MockLogReader.TEST_NAMESPACE, "json",
```

Comment:
Since these calls are not expensive, let's make each call once with empy suppression list, and once with some suppression list.